libraries/wallet/wallet.cpp

```diff
@@ -1440,8 +1440,11 @@ class wallet_api_impl
       optional<asset_object> debt_asset = find_asset(debt_symbol);
       if (!debt_asset)
         FC_THROW("No asset with that symbol exists!");
-      const asset_object& collateral = get_asset(get_object(*debt_asset->bitasset_data_id).options.short_backing_asset);
 
+      //avoid segment error.
+      FC_ASSERT(debt_asset->bitasset_data_id.valid(), "debt_symbol is invalid...");
```

Comment:
Please change error message to something like "Not a bitasset, bidding not possible."

Reply:
thanks. Has been modified.